src/components/views/dialogs/UserPickerDialog.js

```diff
@@ -90,50 +89,7 @@ module.exports = React.createClass({
             inviteList = this._addInputToList();
```

Comment:
There are still things called `invite` and comments mentioning invites in this file